testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -7189,6 +7189,29 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
         "INTEGER ARRAY NOT NULL");
   }
 
+  @Test void testMapContainsKeyFunc() {
```

Comment:
Add more test for 
- NULL
- Complex types
- Empty map
- Type coercion(include function without type coercion)
- Unmatched types

For example:
```sql
map_contains_key(map(null, null), 1);
map_contains_key(map(null, null), null);
map_contains_key(map(array(1), array(2)), array(1));
map_contains_key(map(), 1);
map_contains_key(map(1, 2), cast(1 as double));
map_contains_key(map(1, 2), 'abc');
```

I'm not sure if these examples are valid, you need to give them a try and add more tests.

Reply:
done，thank you